2020-12-03 07:20:53,906 [Thread-0] INFO  checker.TestDatasetVolumeChecker (TestDatasetVolumeChecker.java:testCheckOneVolume(99)) - Executing testCheckOneVolume[null]
2020-12-03 07:20:54,335 [Thread-0] INFO  checker.TestDatasetVolumeChecker (TestDatasetVolumeChecker.java:schedule(182)) - check routine threw exception org.apache.hadoop.util.DiskChecker$DiskErrorException: Fake Exception
2020-12-03 07:20:54,375 [VolumeCheck ResultHandler thread 0] WARN  checker.DatasetVolumeChecker (DatasetVolumeChecker.java:onFailure(380)) - Exception running disk checks against volume Mock for FsVolumeSpi, hashCode: 751758150
org.apache.hadoop.util.DiskChecker$DiskErrorException: Fake Exception
	at org.apache.hadoop.hdfs.server.datanode.checker.TestDatasetVolumeChecker$DummyChecker.schedule(TestDatasetVolumeChecker.java:180)
	at org.apache.hadoop.hdfs.server.datanode.checker.TestDatasetVolumeChecker$DummyChecker.schedule(TestDatasetVolumeChecker.java:171)
	at org.apache.hadoop.hdfs.server.datanode.checker.DatasetVolumeChecker.checkVolume(DatasetVolumeChecker.java:302)
	at org.apache.hadoop.hdfs.server.datanode.checker.TestDatasetVolumeChecker.testCheckOneVolume(TestDatasetVolumeChecker.java:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$StatementThread.run(FailOnTimeout.java:74)
msx-rc 0
